src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -43,7 +43,8 @@ class Connection
     private $consumer;
     private $autoSetup;
     private $maxEntries;
-    private $couldHavePendingMessages = true;
+    private $claimTimeout = 600000;
```

Comment:
I think we should use the same value as the doctrine implementation here https://github.com/symfony/symfony/blob/9f93a38fffd2d57595b2732a3eadeb12d0af0892/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L36 I tshould be configureable the same way.